src/Command/AddMassMediaCommand.php

```diff
@@ -61,8 +63,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $fp = $this->getFilePointer($input, $output);
 
         $delimiter = $input->getOption('delimiter');
+        \assert(null !== $delimiter);
```

Comment:
Feature-request to phpstan-symfony ?

Reply:
Not really, this is a problem with the configure option. If you want to not be able to have nullable as a value you must use VALUE_REQUIRED. 

With VALUE_OPTIONAL (even if you define a default value):

```
❯ console sonata:media:add-multiple --delimiter
$delimiter === null
```

Whit VALUE_REQUIRED:

```
❯ console sonata:media:add-multiple --delimiter
In AddMassMediaCommand.php line 114:

  Please provide a CSV file argument or CSV input stream
```

So Im changing those and the other required values to required instead of optional. Also notice that making the option required doesn't force you to set it IF it has a default value.

Reply:
Fixed.